.circleci/config.yml

```diff
@@ -11,11 +15,62 @@ executors:
     docker:
     - image: circleci/golang:1.13
 
+commands:
```

Comment:
what prevents reusing https://circleci.com/orbs/registry/orb/prometheus/prometheus#commands-publish_images? The `docker` target can be overriden in `Makefile` if we don't need to support multi-arch builds.

Reply:
Well, everything comes from my comment on [`Makefile.common`](https://github.com/prometheus/prombench/pull/320#discussion_r373179288).

When I discussed with @krasi-georgiev , I felt that we choose to use cli tools like `go build` or `promu` as much as possible instead of `make`. That's why `make docker*` are not used completely here. And if we don't use `make`, orb/prometheus (which contains many `make` steps) can be omitted. Plus, when I was new to the project, given by a big `Makefile.common` + `orb` is so scary, it would be fast to complete without too many dependencies. 😈 

Anyhow, I will try `Makefile` way either. But can you @simonpasquier give me a clue on usage of `Makefile.common` in my previous comment ? Will it be dropped ?

Reply:
> Plus, when I was new to the project, given by a big Makefile.common + orb is so scary, it would be fast to complete without too many dependencies. smiling_imp

I agree that it's a bit daunting at first. But they exist for a good reason: they simplify a lot the maintenance work.

> Will it be dropped ?

no the comment you've found predates `Makefile.common`.

Reply:
>  But can you @simonpasquier give me a clue on usage of Makefile.common in my previous comment ? Will it be dropped ?

sorry I should have answered that as well. First the common makefiles was used for building the binaries and it seems that there were some problems with that which led to the development of the promu tool. After that we noticed many common commands for linting and testing go code so decided to share those ammong all repos with yet another common makefile. Now looking at this again it might be a good idea to include as much as possible in the promu tool to reduce the size of the common makefile.